.github/workflows/prettier.yml

```diff
@@ -17,6 +17,13 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       - uses: stefanzweifel/git-auto-commit-action@v4
-        if: ${{ github.event == 'push' || github.event == 'workflow_dispatch' }}
+        if: ${{ github.event_name == 'push' || github.event_name == 'workflow_dispatch' }}
         with:
           commit_message: 'style: format files'
+      - name: Fail CI
+        if: ${{ github.event_name == 'pull_request' || github.event_name == 'pull_request_target' }}
```

Comment:
Are you sure this will work properly? I think there's an action that can make your action fail, not sure though. 🤔

Reply:
This will fail only if the PR has errors 🙂